addons/arsenal/functions/fnc_onArsenalOpen.sqf

```diff
@@ -296,6 +296,10 @@ if (is3DEN) then {
 } else {
     GVAR(centerNotPlayer) = (GVAR(center) != player);
 
+    if (currentVisionMode ACE_Player == 1) then {
+        GVAR(center) action ["NVGogglesOff", GVAR(center)];
```

Comment:
Should this line also use `ace_player`?

Reply:
both are the same in that case, it doesn't really matter outside readability